setuptools/_vendor/autocommand-2.2.2.dist-info/METADATA

```diff
@@ -320,7 +320,7 @@ optional arguments:
 STOP and STEP default to 1
 ```
 
-Even though autocommand is being applied to the `wrapper` returned by `print_yielded`, it still retreives the signature of the underlying `seq` function to create the argument parsing.
+Even though autocommand is being applied to the `wrapper` returned by `print_yielded`, it still retrieves the signature of the underlying `seq` function to create the argument parsing.
```

Comment:
We should not modify files in the `_vendor` and `_distutils` directly via PRs to setuptools. Instead it is better to send the PRs to the upstream repositories.